### ISSUE TITLE
Test: Issue Reporter Command imrpovements

### ISSUE THREAD

**Issue by justschen** [microsoft/vscode#203094]
Continuation of https://github.com/microsoft/vscode/issues/196324

- [ ] anyOS 
- [ ] anyOS 
- [ ] anyOS 

Complexity: 4

[Create Issue](https://github.com/microsoft/vscode/issues/new?body=Testing+%23180757%0A%0A&assignees=justschen)

---

## Overview
Added improvements to the Issue Reporter Command. Instead of an API, we now are testing a basic version of the `openIssueReporter` command that does virtually the same thing as the API.

The following is the structure of what an example call to `openIssueReporter` could look like.
```ts
export function openIssueReporter() {
	vscode.commands.executeCommand('workbench.action.openIssueReporter', {
		extensionId: 'ms-python.python',
		issueTitle: 'title here',
		command: {
			template: templateString,
			data: dataString,
			uri: Uri.parse('https://github.com/microsoft/vscode-copilot-release').toString(), // changed url here for demo
		}
	});
}
```

note: `template`, `data`, and `uri` are all expected to be `string` typed.

The general idea is that from anywhere, you could run this command and give it some parameters, mainly things like a template (which will go to the body), data (which if contributed, will add a data field), or a uri (which if contributed, will replace the repo url).

## To test
1. Implement the command. This can be done basically anywhere, but for simplicity, an extension may be the easiest. If you have a personal extension to test on, fantastic. If not, check out [this extension's implementation](https://github.com/Yoyokrazy/clean-nb-imports-ext/blob/74f23aabe58925270d81906d3d1ddfcd40b84452/src/extension.ts#L17). 
2. Once the command has been implemented, try it out by running the command, which should open the issue reporter, and see if the correct things are in the issue reporter window. 
3. Try contributing all of the commands, some of the commands, or none at all to see the flow.
4. Ensure that the original issue reporter workflow (clicking on `Help: Report Issue` from the command palette) is not broken.

## Some things to make sure:
* The right things are hidden or shown
* When you switch between an extension and another extension, the correct things are shown, and correct url (when submitting) is reached.
* Data is populated correctly in both template and extension data sections.
* Correct text is sent to GitHub after pressing submit
 

Thanks!

**Comment by TylerLeonhardt** [comment]
I don't think this replaces the previous IssueDataProvider. The IssueDataProvider was invoked when you opened the Issue Reporter and chose the extension. This, on the other hand, needs to be invoked by an extension.

With only this command, as a user, I can still open the Issue reporter, go to the extension with the issue and not see any of the auto-filled content.

We need a solution that covers both of these scenarios (invoked by extension, and by issue reporter)
